### DISCUSSION TITLE
Cachet 3.x — January Update

### DISCUSSION THREAD

**Discussion by jbrooksuk** [cachethq/cachet#4370] [Announcements]
Hey all,

I've just published a new update on the progress of Cachet 3.x. You can read that on our blog at https://blog.cachethq.io/cachet-development-update-jan-2024/

**Comment by AdoDreamtilt** [comment]
Hi, how are things progressing with this project?

**Reply by jbrooksuk** [reply]
Hey @AdoDreamtilt! We're still progressing well. I posted a blog post at the end of February showing everything the v3 demo can now do. https://blog.cachethq.io/cachet-v3-demo/
